queries.sql

```diff
@@ -8,3 +8,31 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name <> 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
```

Comment:
- [ ] Great job so far 👏🏻 However you're missing to insert these transactions 👇🏻 Kindly Let's address this. ✍🏻 

![image](https://user-images.githubusercontent.com/79058364/235930665-85234c32-85db-46c2-86f9-e05a5aec1560.png)